### PR TITLE
Modified README.md. Added HOW_TO_USE.md and DEPLOY.md

### LINKED ISSUE #36: Modify README.md
Moving `How to use` and `Deploy on GitHub pages` to separate `.md` files

`DEPLOY.md`
`HOW_TO_USE.md`

### PR DESCRIPTION
**Pull request by maxmart97**
Fixes #36 
-------
As per issue, I separated the "How to use" and "Deploy on github pages" sections from the main README.md and put these sections into new files.

### CODE REVIEW

**Comment by maxmart97** [comment]
I removed the page anchors at first because it didn't make sense to me to have a page anchor when there is only one section of text on the page. I didn't know that removing them would also shrink the font and remove the underline. Since that occurred, I added the page anchors back. The files should be in good shape.

File: README.md [hunk 1]
```diff
@@ -3,33 +3,6 @@
 This is a simple profile card app build in ReactJs. It uses a JSON file for managing data. Data is fetched from the JSON file and it is shown in the UI.
 
 ![alt text](assets/img.png "Profile Card Example")
```
Comment (arshadkazmi42) [inline README.md]: Can we add a link to the `HOW_TO_USE.md` and `DEPLOY.md` in `README.md`

**Comment by arshadkazmi42** [comment]
Congrats on landing you first PR :tada:

**Comment by maxmart97** [comment]
@arshadkazmi42 Thank you very much for letting me work on this "issue." I know it was simple and could have taken you 5 minutes, but I appreciate you letting me do it so that I could figure out how to properly do a pull request. I also learned how to add links in .md files, which is cool. I hope to contribute actual code to open source projects in the future.

**Pull request merged**